pkg/util/kubernetes/kubelet/kubelet_interface.go

```diff
@@ -0,0 +1,31 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+// +build kubelet,!orchestrator
```

Comment:
In what cases would we want to build with the `kubelet` tag and exclude `orchestrator`?

Reply:
in all cases expect the process agent for now to avoid increasing the binary size 😅
the need for `dogstatsd` that uses the `kubelet` tag for the tagger, pulling the extra k8s dependencies is making the static binary size blowup from ~17mb to ~25mb (making the integration test on the dsd binary failing)
https://github.com/DataDog/datadog-agent/blob/d6fcfd3549ec2c6e89a56a1aa7dad04bdcf3c575/tasks/dogstatsd.py#L24